kuma/static/js/lang-switcher.js

```diff
@@ -0,0 +1,58 @@
+(function (win, doc, $) {
+    function storeLocaleChange(code, name) {
+        localStorage.setItem('changed-locale-to', JSON.stringify({code: code, name: name}));
+    }
+
+    if(win.mdn.features.localStorage) {
+        var langSwitcherSelector = document.getElementById('language'),
+            langSwitcherButton = document.getElementById('translations');
```

Comment:
Code style: Implicitly define each `var` on it's own line. Do not use the single `var` pattern

Reply:
@schalkneethling I was wondering the reason behind not using single `var` pattern. In my early days, some people suggested me to use this pattern while writing javascript.
Searching in internet did not provide me any actual answer.
Can you please explain more about the underlying reason behind it?
I have found this on eslint documentation https://eslint.org/docs/rules/one-var

Reply:
For the most part it is a stylistic choice. One of the other reasons is that it avoid accidental implicit globals, should one forget the `var`. Basically one is more explicit about your definitions. Now, with the additional of `let`, `const` etc. being explicit is even more important as it suggests intent.

Our `eslint` rules are now also set to fail on the single var pattern. I used to also like the single `var` pattern but, over time I have come to prefer being explicit by clearly defining each variable.

Reply:
@schalkneethling I understand and also accept the concern you have raised. Since we are using ES5 still, I thought it should be one var convention.
But as we are hoping to move writing to ES6 code, I think its better approach to not use the one var constrains.

Regarding eslint rules, we use [`one-var-declaration-per-line`](https://github.com/mozilla/kuma/blob/master/.eslintrc.js#L50) that does not fail on single var pattern and thats why my PR passes the current eslint. Do you want I add the rule to fail `one-var` pattern in `.eslintrc.js`?